src/admin/content/views/ContentDetail.tsx

```diff
@@ -96,6 +101,10 @@ const ContentDetail: FunctionComponent<ContentDetailProps> = ({ history, match,
 						componentState={contentBlockConfig.components.state}
 						contentWidth={get(content, 'content_width')}
 						blockState={contentBlockConfig.block.state}
+						history={history}
```

Comment:
Mss gwn best in de ContentBlockPreview `withRouter` gebruiken, dan moet je die niet altijd doorgeven.
Het user object wordt ook niet gebruikt zover ik zien in de Preview.